src/database.rs

```diff
@@ -96,7 +103,18 @@ impl Database {
                 let uuid = parse_uuid(filter_value)?.to_string();
                 query_builder.push_bind(uuid);
             }
+            "place_ids" => {
+                query_builder.push("metadata->>'placeId' = ANY(");
+                let place_ids = filter_value
+                    .split(',')
+                    .map(|s| s.trim().to_string())
+                    .collect::<Vec<String>>();
+                tracing::info!("Parsed place IDs for query: {:?}", place_ids);
+                query_builder.push_bind(place_ids);
```

Comment:
Does this already join them with `,` or should we modify the split we are doing to re-add `,` between each place id for the query?